### PR TITLE
super singularity god super singularity

### PR DESCRIPTION
**Pull request by SonixApache**
Adds another level of singularity; stage 7(13).
Happens when you throw a full crystal at a super singulo.
It shits out supermatter seas!

I added this because when you throw a full crystal at a singularity, it just turns it super.
Shards already do that when it's at stage 5.
The crystal would explode, but the singulo would eat the supermatter, resulting in a galaxy-end event without any supermatter sea nonsense.
This PR is to adjust that.

Also contains some minor fixes like not throwing errors when a super sing eats crystals.
SSGSS
Sprite could use work, I just did a simple color swap of the stage 6(11). Doesn't seem to be any takers, though.

🆑 
 - rscadd: Recent developments in black hole and supermatter research have confirmed the possibility of an even more destructive entity than the fabled 'Super Singularity'. Please keep supermatter crystals and 'Super Singulos' separated at all times.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
this nigga going for stellaris mod tier reskins hahaha lmfao

**Comment by Kurfursten** [comment]
Does it actually need to be bigger-er? I mean really the alternate color and supermatter sea seems plenty

The problem with really really big stuff is that it interacts a bit oddly with eating shit because of corners, and it was already really noticeable at 11

**Comment by SonixApache** [comment]
~~The color change is probably enough, yeah. I'll turn it back down as soon as I get a better sprite.~~
Code-wise, it has to be bigger, for the sake of not having to deal with a million extra vars.
I'll leave the sizes as is, but keep the consume range the same as the super sing, so you'd have to get 'closer' for it to eat you.

File: code/modules/power/singularity/singularity.dm [hunk 1]
```diff
@@ -629,6 +666,8 @@
 */ //Fuck you centcomm
 
 /obj/machinery/singularity/Destroy()
+	consume(555)
```
Comment (MadmanMartian) [inline code/modules/power/singularity/singularity.dm]: What is this for?

Comment (SonixApache) [reply code/modules/power/singularity/singularity.dm]: Hm, probably left over testing from when the supersea was eating my singulos. I'll rip it out.

**Pull request merged**